app/app.go

```diff
@@ -561,35 +559,43 @@ func NewStrideApp(
 	)
 	icaModule := ica.NewAppModule(&app.ICAControllerKeeper, &app.ICAHostKeeper)
 
-	// Create the middleware stacks
+	// Define the IBC stacks
+	// Stack one (ICAHost Stack) contains:
+	// - IBC
+	// - ICAHost
+	// - base app
+	icaHostIBCModule := icahost.NewIBCModule(app.ICAHostKeeper)
 
-	// Stack one contains
+	// Stack two (ICAController Stack) contains
 	// - IBC
-	// - ICA
-	// - icacallbacks
+	// - ICAController (middleware)
+	// - icacallbacks (middleware)
 	// - stakeibc
 	// - base app
-	var icamiddlewareStack porttypes.IBCModule
-	icamiddlewareStack = icacallbacksmodule.NewIBCModule(app.IcacallbacksKeeper, stakeibcIBCModule)
-	icamiddlewareStack = icacontroller.NewIBCModule(app.ICAControllerKeeper, icamiddlewareStack)
-	icaHostIBCModule := icahost.NewIBCModule(app.ICAHostKeeper)
+	icaCallbacksStack := icacallbacksmodule.NewIBCModule(app.IcacallbacksKeeper, stakeibcIBCModule)
+	icaControllerStack := icacontroller.NewIBCModule(app.ICAControllerKeeper, icaCallbacksStack)
 
-	// Stack two contains
+	// Stack three (Records Stack) contains
```

Comment:
We should probably call this the transfer stack, for clarity

Reply:
my thinking was we should label each stack from either the top or bottom but be consistent about it. Which would mean we either do ICAControllerStack and RecordsStack, or StakeibcStack and TransferStack.

I thought using the top of the stack made the code slightly cleaner since you define them bottom up. What do you think?